README.md

```diff
@@ -1,5 +1,11 @@
 # simple-react-calendar <sup>[![Version Badge][npm-version-svg]][package-url]</sup>
 
+---
+
+## ⚠️ Please note that the repository is no longer maintained and was archived. If you need to use a calendar, please use the `Calendar` component from `@toptal/picasso` package or [React Day Picker](https://react-day-picker.js.org/) library.
```

Comment:
Maybe makes sense to not mention `@toptal/picasso` because if we move calendar out into separate package, this message will be forgotten and will be wrong.

Reply:
@OleksandrNechai Thank you for the suggestion! I assume that if it resides in a separate package, it will still be a part of Picasso library, so what do you think about

> Please note that the repository is no longer maintained and was archived. If you need to use a calendar, please use the `Calendar` component from [Picasso component](https://picasso.toptal.net/) library or [React Day Picker](https://react-day-picker.js.org/) library

Reply:
The https://toptal-core.atlassian.net/browse/FX-4061 was added to expose the `Calendar` calendar in Storybook

Reply:
The message was updated, thank you!